airbyte-integrations/connectors/source-shopify/source_shopify/http_request.py

```diff
@@ -21,6 +21,8 @@
     exceptions.SSLError,
 ) + RESPONSE_CONSUMPTION_EXCEPTIONS
 
+_NO_ERROR_RESOLUTION = ErrorResolution(ResponseAction.SUCCESS, None, None)
```

Comment:
Can we add the comment about this, what it stands for and what is the parrent used?